tracer/src/Datadog.Trace/AppSec/Waf/Initialization/WafConfigurator.cs

```diff
@@ -101,7 +107,21 @@ private static void LogRuleDetailsIfDebugEnabled(JToken root)
             return File.OpenRead(rulesFile);
         }
 
-        internal static JToken? DeserializeEmbeddedRules(string? rulesFilePath)
+        internal static JToken? DeserializeSchemaExtractionConfig()
```

Comment:
It's definitely a nit, and not for this PR, but given we have this json file at compile time, it would be nice to use it to power a source generator, so we don't have to do all this deserialization work at startup 🤔

Reply:
that's interesting, I'll create a backlog for both the schema and rules, thank you :)